code-challenges.test.js

```diff
@@ -13,6 +13,13 @@
 // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
 // a) Create a test with an expect statement using the variable provided.
+// describe("cappy", () => {
+//     it("Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized, () => {
+//          const hitchhikersCharacters = [{ name: "ford prefect", occupation: "a hitchhiker" },{ name: "zaphod beeblebrox", occupation: "president of the galaxy" },{ name: "arthur dent", occupation: "a radio employee" }]
+//          Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+//          expect(cappy = (hitchhikersCharacters)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
```

Comment:
Remember that your expect statement should be performing a function call so it can compare the output of the function with the expected output.